test/Azure.Functions.Cli.Tests/E2E/StartTests_default.runsettings

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RunSettings>
+  <RunConfiguration>
+    <TestCaseFilter>(Category != RequiresNestedInProcArtifacts)</TestCaseFilter>
+  </RunConfiguration>
```

Comment:
curious why we're using .runsettings when the filters can be set via dotnet test --filter? Is this just easier because of how the build.csproj works?

Reply:
yeah so I changed it to use .runsettings since it might be confusing for users once they are running the tests locally (using test explorer or running `dotnet test` themselves). Matthew talked about this in the comment here: https://github.com/Azure/azure-functions-core-tools/pull/3841#discussion_r1778854154